### ISSUE TITLE
Summoning The Darkness from hand or deck triggers Jungle Giants

### ISSUE THREAD

**Issue by Kybxd** [HearthSim/hs-bugs#871]
* https://youtu.be/TrDljuLqM_M

Faceless Manipulator and Moat Lurker don't trigger Jungle Giants. I guess the difference is that they create a new entity, while Dirty Rat and Y'Shaarj only move it to board.

**Comment by riQQ** [comment]
Why should this be a bug? Jungle Giants text says 
> Summon 5 minions with 5 or more attack.
It doesn't say play / summon from hand.

**Comment by Kybxd** [comment]
Beacuse Jungle Giants doesn't trigger when you play it, copy it by Faceless Manipulator or summon it from Moat Lurker's deathrattle.

**Comment by troggnostupidhs** [comment]
@Kybxd the inconsistency sounds like the bug

**Comment by ruthnaebeck** [comment]
If the darkness is played from hand by the player, Jungle Giants does not get count completion:
https://youtu.be/8hD1CVEi9Wc

The count completion may be happening for recruit / pulling mechanisms because for 1 second The Darkness is a 20/20 minion on board then becomes dormant.

I agree this is inconsistent behavior and sounds like a bug.

**Comment by troggnostupidhs** [comment]
https://playhearthstone.com/en-us/blog/21802981

> * Summon and Play triggers are now evaluated using their in-hand stats before they are affected by board modifiers and their Battlecry. 
>   * Ex. Playing Faceless Manipulator targeting a 5-attack minion will no longer advance Jungle Giants.  It will be considered a 3/3.
> 
> * Minions that enter the battlefield without being played from hand will have any (Whenever or After) summon triggers evaluated before board modifiers are applied.
>   * Ex. If Azure Drake is pulled onto the battlefield adjacent to Dire Wolf Alpha it will no longer advance Jungle Giants.
> 
> * Choose One minions that Transform now consider their transformation effects before entering play. 
>   * Ex. Playing Shellshifter and selecting the 5/3 option will count as a 5-attack minion and advance Jungle Giants.

**Comment by troggnostupidhs** [comment]
Can anyone test?

**Comment by Kybxd** [comment]
Only test one situation. Recruiting The Darkness from deck still advances Jungle Giants after Patch 11.2 update.

**Comment by PattuX** [comment]
Do you know the transform timing? When playing it, it seems like it transforms in hand already (as you mentioned in #970 because of the similarity with the Glinda bug) but I have no clue for summoning it.

**Comment by ruthnaebeck** [comment]
Confirming Kybxd's statement:
> Recruiting The Darkness from deck still advances Jungle Giants after Patch 11.2 update.

Video Evidence - https://youtu.be/n-SXC-fqoNs

**Comment by Jetz72** [comment]
Isn't the dormant token technically a 20/20? It's hidden on the board as part of its Untouchable status, but mousing over it still reveals the stats.

**Comment by Kybxd** [comment]
> {"artist":"L. Lullabi & K. Turovec","attack":20,"cardClass":"NEUTRAL","cost":4,"dbfId":48639,"elite":true,"health":20,"hideStats":true,"id":"LOOT_526d","mechanics":["IGNORE_HIDE_STATS_FOR_BIG_CARD","TRIGGER_VISUAL","UNTOUCHABLE"]

Yes it is 20/20. But this cannot explain the inconsistency of summoning from hand/deck and other summonings (which create a new entity).

**Comment by troggnostupidhs** [comment]
https://playhearthstone.com/en-us/blog/22546493
> The Darkness no longer advances Jungle Giants when summoned.
